### ISSUE TITLE
Package doesn't work when installed in custom folder (pip's --target)

### ISSUE THREAD

**Issue by jack1142** [nielstron/quantulum3#111]
**Describe the bug**
When package quantulum3 is installed into custom folder (e.g. "lib/"), it tries to load `units.json` file from wrong directory which causes an error.

**To Reproduce**
1. Install package into custom folder (e.g. "lib/")
```
pip3 install --upgrade --target lib quantulum3
```
2. From main folder (the folder where lib folder is) run this code:
```
import sys
sys.path.insert(0, 'lib')
from quantulum3 import parser

parser.parse('test string')
```
3. See error
```
Traceback (most recent call last):
  File "lib\quantulum3\load.py", line 34, in cached_function
    return _CACHE_DICT[id(funct)][lang]
KeyError: 1609576810016

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "lib\quantulum3\load.py", line 34, in cached_function
    return _CACHE_DICT[id(funct)][lang]
KeyError: 1609576777792

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "test.py", line 5, in <module>
    parser.parse('test string')
  File "lib\quantulum3\parser.py", line 444, in parse
    for item in reg.units_regex(lang).finditer(text):
  File "lib\quantulum3\load.py", line 36, in cached_function
    result = funct(lang)
  File "lib\quantulum3\regex.py", line 324, in units_regex
    list(load.units(lang).surfaces.keys()) + list(
  File "lib\quantulum3\load.py", line 36, in cached_function
    result = funct(lang)
  File "lib\quantulum3\load.py", line 328, in units
    return Units(lang)
  File "lib\quantulum3\load.py", line 229, in __init__
    with path.open(encoding='utf-8') as file:
  File "C:\Users\me\AppData\Local\Programs\Python\Python35\lib\pathlib.py", line 1151, in open
    opener=self._opener)
  File "C:\Users\me\AppData\Local\Programs\Python\Python35\lib\pathlib.py", line 1005, in _opener
    return self._accessor.open(self, flags, mode)
  File "C:\Users\me\AppData\Local\Programs\Python\Python35\lib\pathlib.py", line 371, in wrapped
    return strfunc(str(pathobj), *args)
FileNotFoundError: [Errno 2] No such file or directory: 'lib\\quantulum3\\units.json\\lib\\quantulum3\\_lang\\en_US\\units.json'
```

**Expected behavior**
Package shouldn't have troubles with loading.

**Additional information:**
 - Python Version: 3.5
 - Classifier activated/ sklearn installed: no
 - OS: probably all, but tested on Windows 10 1803 and Ubuntu 18.04.1 LTS
 - Version: 0.6.5

**Comment by nielstron** [comment]
Thanks for the feedback!
> ```
>FileNotFoundError: [Errno 2] No such file or directory: 'lib\\quantulum3\\units.json\\lib\\quantulum3\\_lang\\en_US\\units.json'
>```

Fascinating, somehow the path is added twice here. I will have a look into it.

**Comment by nielstron** [comment]
Did you try out what happened when it was *not* installed into a custom folder?

**Comment by jack1142** [comment]
Yes, it works without a scratch then.

**Comment by nielstron** [comment]
Gotcha: 
https://github.com/nielstron/quantulum3/blob/593e2436172e7b7ebbce6bf71051e65fe46c7099/quantulum3/load.py#L240

This should long have thrown errors, it is fascinating that no one has yet caught on to this...

**Comment by nielstron** [comment]
Done. This should be fixed from version 0.6.6. Thanks for pointing this out!

**Comment by dzombiee** [comment]
Describe the bug
When running parser.parse, it outputs KeyError: 0 randomly, meaning for a string it works fine, but for a different string gives KeyError: 0

To Reproduce
Steps to reproduce the behavior:
1. install the package
`!pip install quantulum3`

2. import the parser from quantulum3 and run for a string

`from quantulum3 import parser`
`parser.parse('a string with 9 liters')`

4. see error
```KeyError                                  Traceback (most recent call last)
File ~\Anaconda3\lib\site-packages\quantulum3\load.py:33, in cached.<locals>.cached_function(lang)
     32 try:
---> 33     return _CACHE_DICT[id(funct)][lang]
     34 except KeyError:

KeyError: 2068417855552

During handling of the above exception, another exception occurred:

KeyError                                  Traceback (most recent call last)
Input In [7], in <cell line: 1>()
----> 1 parser.parse('a string with 9 liters')

File ~\Anaconda3\lib\site-packages\quantulum3\parser.py:517, in parse(text, lang, verbose)
    514 try:
    515     uncert, values = get_values(item, lang)
--> 517     unit, unit_shortening = get_unit(item, text)
    518     surface, span = get_surface(shifts, orig_text, item, text, unit_shortening)
    519     objs = build_quantity(
    520         orig_text, text, item, values, unit, surface, span, uncert, lang
    521     )

File ~\Anaconda3\lib\site-packages\quantulum3\parser.py:345, in get_unit(item, text, lang)
    343     if unit:
    344         unit_surface, power = parse_unit(item, unit, slash, lang)
--> 345         base = dis.disambiguate_unit(unit_surface, text, lang)
    346         derived += [{"base": base, "power": power, "surface": unit_surface}]
    348 unit = get_unit_from_dimensions(derived, text, lang)

File ~\Anaconda3\lib\site-packages\quantulum3\disambiguate.py:18, in disambiguate_unit(unit_surface, text, lang)
     13 """
     14 Resolve ambiguity between units with same names, symbols or abbreviations.
     15 :returns (str) unit name of the resolved unit
     16 """
     17 if clf.USE_CLF:
---> 18     base = clf.disambiguate_unit(unit_surface, text, lang).name
     19 else:
     20     base = (
     21         load.units(lang).symbols[unit_surface]
     22         or load.units(lang).surfaces[unit_surface]
     23         or load.units(lang).surfaces_lower[unit_surface.lower()]
     24         or load.units(lang).symbols_lower[unit_surface.lower()]
     25     )

File ~\Anaconda3\lib\site-packages\quantulum3\classifier.py:258, in disambiguate_unit(unit, text, lang)
    255     return load.units(lang).names.get("unk")
    257 if len(new_unit) > 1:
--> 258     transformed = classifier(lang).tfidf_model.transform([clean_text(text, lang)])
    259     scores = classifier(lang).classifier.predict_proba(transformed).tolist()[0]
    260     scores = zip(scores, classifier(lang).target_names)

File ~\Anaconda3\lib\site-packages\quantulum3\load.py:35, in cached.<locals>.cached_function(lang)
     33     return _CACHE_DICT[id(funct)][lang]
     34 except KeyError:
---> 35     result = funct(lang)
     36     _CACHE_DICT[id(funct)] = {lang: result}
     37     return result

File ~\Anaconda3\lib\site-packages\quantulum3\classifier.py:213, in classifier(lang)
    206 @cached
    207 def classifier(lang="en_US"):
    208     """
    209     Cached classifier object
    210     :param lang:
    211     :return:
    212     """
--> 213     return Classifier(lang=lang)

File ~\Anaconda3\lib\site-packages\quantulum3\classifier.py:189, in Classifier.__init__(self, obj, lang)
    187     path = language.topdir(lang).joinpath("clf.joblib")
    188     with path.open("rb") as file:
--> 189         obj = joblib.load(file)
    191 cur_scipy_version = pkg_resources.get_distribution("scikit-learn").version
    192 if cur_scipy_version != obj.get("scikit-learn_version"):  # pragma: no cover

File ~\Anaconda3\lib\site-packages\joblib\numpy_pickle.py:577, in load(filename, mmap_mode)
    575     filename = getattr(fobj, 'name', '')
    576     with _read_fileobject(fobj, filename, mmap_mode) as fobj:
--> 577         obj = _unpickle(fobj)
    578 else:
    579     with open(filename, 'rb') as f:

File ~\Anaconda3\lib\site-packages\joblib\numpy_pickle.py:506, in _unpickle(fobj, filename, mmap_mode)
    504 obj = None
    505 try:
--> 506     obj = unpickler.load()
    507     if unpickler.compat_mode:
    508         warnings.warn("The file '%s' has been generated with a "
    509                       "joblib version less than 0.10. "
    510                       "Please regenerate this pickle file."
    511                       % filename,
    512                       DeprecationWarning, stacklevel=3)

File ~\Anaconda3\lib\pickle.py:1212, in _Unpickler.load(self)
   1210             raise EOFError
   1211         assert isinstance(key, bytes_types)
-> 1212         dispatch[key[0]](self)
   1213 except _Stop as stopinst:
   1214     return stopinst.value

KeyError: 0```

Additional information:

    Python Version: 3.9
    Classifier activated/ sklearn installed: yes
    OS: tested on Windows 11
    Version: 0.8.1
